lib/text_client.rb

```diff
@@ -0,0 +1,22 @@
+require "twilio-ruby"
+require "./lib/message"
+
+class TextClient
+
+  attr_reader :message_formatter
+
+  def initialize(message_formatter = Message.new)
+    @message_formatter = message_formatter
+    @account_sid = ENV["TWILIO_ACCOUNT_SID"]
+    @auth_token = ENV["TWILIO_AUTH_TOKEN"] 
+    @client = Twilio::REST::Client.new(@account_sid, @auth_token)
+  end
+  
+  def send_text
```

Comment:
Good way to achieve abstraction you have successfully built send_text and have hid this from the users perspective. For a user ordering they simply receive a text without doing anything. - You interact with your order by using only a few buttons. What’s going on under the hood? You don’t have to know — implementation details are hidden. You only need to know a short set of actions. 

ref: https://www.freecodecamp.org/news/object-oriented-programming-concepts-21bb035f7260/